modules/javafx.graphics/src/main/java/javafx/css/Selector.java

```diff
@@ -58,31 +58,67 @@ static Selector getUniversalSelector() {
     }
 
     private Rule rule;
+    /**
+     * Sets the {@code Rule} of this Selector.
+     * @param rule the {@code Rule} of this Selector
+     */
     void setRule(Rule rule) {
         this.rule = rule;
     }
+
+    /**
+     * Gets the {@code Rule} of this Selector.
+     * @return rule
+     */
     public Rule getRule() {
         return rule;
     }
 
     private int ordinal = -1;
+    /**
+     * Sets the ordinal of this Selector.
+     * @param ordinal the ordinal of this Selector
+     */
     public void setOrdinal(int ordinal) {
         this.ordinal = ordinal;
     }
+
+    /**
+     * Gets the ordinal of this Selector.
+     * @return the ordinal of this Selector
+     */
     public int getOrdinal() {
         return ordinal;
     }
 
+    /**
+     * Creates a {@code Match}.
+     * @return match
+     */
     public abstract Match createMatch();
 
-    // same as the matches method expect return true/false rather than a match
+    /**
+     * Gets whether this {@code Selector} applies to the given {@code Styleable}.
+     * It is the same as the {@link createMatch} method except it returns a boolean rather than a {@code Match}.
```

Comment:
I see you changed this from `matches` to `createMatches`, since there is no `matches` method. However, I now wonder if this comment is correct in the first place. The `createMatch` method doesn't take a `Styleable` so it seems unlikely that it would be semantically equivalent to calling `createMatches`. Unless you can figure out what was meant by the original comment, it might best to just remove it.

Reply:
It is out of place comment. I think, it is better to remove it.